src/main/java/uk/gov/justice/hmpps/prison/service/PrisonerCreationService.java

```diff
@@ -64,12 +64,19 @@ public String createPrisoner(final RequestToCreate requestToCreate) {
                     throw new BadRequestException(format("Prisoner with PNC %s already exists with ID %s", longPnc, identifier.getOffender().getNomsId()));
                 });
 
+        } else if (StringUtils.isNotBlank(requestToCreate.getCroNumber())) {
```

Comment:
Should we check the CRO even if there is a PNC?

Reply:
duh yes!